apps/docs/src/.vitepress/browser-loader.data.ts

```diff
@@ -0,0 +1,72 @@
+import fs from "fs";
+import https from "https";
+import path, { dirname } from "path";
+import { fileURLToPath } from "url";
+import { defineLoader } from "vitepress";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const browserslistRcPath = path.resolve(__dirname, "../../../../.browserslistrc");
```

Comment:
```suggestion
const browserslistRcPath = fileURLToPath(new URL("../../../../.browserslistrc", import.meta.url));
```

Usually we do this like this so we save 2 lines :)